composer.json

```
@@ -26,17 +26,28 @@
         }
     ],
     "require":{
-        "php": ">=5.4",
-        "contao/core-bundle": "^3.5 || ^4.4",
-        "contao-community-alliance/composer-plugin": "^2.4 || ^3.0",
-        "jean85/pretty-package-versions": "^1.0",
-        "composer/semver": "^1.0"
+        "php": "^7.1 || ^8.0",
+        "contao/core-bundle": "^4.9",
+        "contao/news-bundle": "^4.9",
+        "doctrine/dbal": "^2.10",
+        "symfony/dependency-injection": " ^4.4 || ^5.1",
+        "symfony/config": "^4.4 || ^5.1",
+        "symfony/http-kernel": "^4.4 || ^5.1"
     },
-    "extra":{
-        "contao":{
-            "sources": {
-                "system/modules/news_sorting": "system/modules/news_sorting"
-            }
+    "require-dev": {
+        "contao/manager-plugin": "^2.11",
+        "codefog/contao-news_categories": "^3.0"
+    },
+    "conflict": {
+        "contao/manager-plugin": "<2.11 || >= 3.0",
+        "codefog/contao-news_categories": "<3.0 || >= 4.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "NewsSortingBundle\\": "src/"
```

```suggestion
            "InspiredMinds\\NewsSortingBundle\\": "src/"
```

Minor nitpick: I would prefer the namespace prefix

```
InspiredMinds\ContaoNewsSorting\
```

in order to be in line with (most) of our other extensions. The namespace in the class files themselves need to be adjusted as well of course.